packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -40,7 +40,9 @@ export interface TCGCContext {
   flattenUnionAsEnum?: boolean;
   arm?: boolean;
   referencedTypeMap: Map<Type, SdkModelType | SdkEnumType | SdkUnionType | SdkNullableType>;
-  referencedPropertyMap: Map<ModelProperty, SdkModelPropertyType>;
+  modelPropertyMap: Map<ModelProperty, SdkModelPropertyType>;
```

Comment:
I understand `referencedPropertyMap` is renamed to `modelPropertyMap`? If so, we could add it in changelog so that language emitters could migrate more smoothly.

Reply:
Does any emitter uses it? Or should this rather be an implantation detail (as `ModelProperty` is TypeSpec type)?

Reply:
this is not for emitter to use. let me rename them with `__`.